airbyte-integrations/bases/base-java/src/main/java/io/airbyte/integrations/destination_async/DetectStreamToFlush.java

```diff
@@ -122,15 +138,13 @@ Optional<StreamDescriptor> getNextStreamToFlush(final long queueSizeThresholdByt
    * This method also returns debug string with info that about the computation. We do it this way, so
    * that the debug info that is printed is exactly what is used in the computation.
    *
-   * @param stream stream
+   * @param latestFlushTimeMs latestFlushTimeMs
    * @return is time triggered and a debug string
    */
   @VisibleForTesting
-  ImmutablePair<Boolean, String> isTimeTriggered(final StreamDescriptor stream) {
-    final Boolean isTimeTriggered = bufferDequeue.getTimeOfLastRecord(stream)
```

Comment:
how does the new behavior differ from the old behavior? (I haven't looked at what `getTimeOfLastRecord` was doing)

it looks like the previous code was also trying to avoid records sitting for longer than 5 minutes?

Reply:
The old record was triggerring the flush if there was no record during at least 5 minutes. Now we force a sync to trigger at least every 5 minutes, wether there were a record or not.

Reply:
aha. that makes sense. I think MAX_TIME_BETWEEN_REC_MIN  is now unused and can be removed? but otherwise lgtm

Reply:
Indeed, done.